### PR TITLE
Digest days

### PR DESCRIPTION
**Pull request by dirkkelly**

### CODE REVIEW

File: lib/digest_helper.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    def today_as_digest_day
+      Date.today.strftime('%A').downcase
+    end
+
+    def digest_day?
+      digest_day == today_as_digest_day
+    end
+
+    def send
```
Comment (bigfleet) [inline lib/digest_helper.rb]: nitpick, but let's not override send

Comment (dirkkelly) [inline lib/digest_helper.rb]: Cheers, good call

**Comment by dirkkelly** [comment]
@bigfleet renamed method, thanks for that

File: lib/digest_helper.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      ::Citygram::Workers::Notifier
+      ::Subscription.notifiables.where(channel: 'email').paged_each do |subscription|
+        if subscription.has_events?
+          ::Citygram::Workers::Notifier.perform_async(subscription.id, nil)
+        end
+      end
+    end
+
+    def send_notifications_if_digest_day
+      send if digest_day?
```
Comment (bigfleet) [inline lib/digest_helper.rb]: Still calling send?

File: spec/lib/digest_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe Citygram::DigestHelper do
+  describe "send_notifications_if_digest_day" do
+    context 'when digest_day is today' do
+      it 'sends the digest' do
+        allow_any_instance_of(Citygram::DigestHelper).to receive(:digest_day?).and_return(true)
+        expect_any_instance_of(Citygram::DigestHelper).to receive(:send)
```
Comment (bigfleet) [inline spec/lib/digest_helper_spec.rb]: Still testing send? (x2 on line 17)

**Comment by dirkkelly** [comment]
Thanks for all the feedback @bigfleet, sorry about the back and forth.

We're testing this in production tomorrow morning.

screen shot 2015-07-22 at 10 42 17 pm
screen shot 2015-07-22 at 10 42 06 pm
screen shot 2015-07-22 at 10 43 17 pm

We'll let you know how it goes!

**Comment by dirkkelly** [comment]
This ran successfully in production!

**Pull request merged**